### ISSUE TITLE
Using [[toc]] causes hydration mismatch

### ISSUE THREAD

**Issue by DawidBlaszczyk** [vuejs/vitepress#1313]
### Describe the bug

Adding [[toc]] to the docs results in `Hydration completed but contains mismatches.` error in the browser.

### Reproduction

1. Create doc with [[toc]]
```
[[toc]]

## test

test
```
2. Run `vitepress build docs`
3. Run `vitepress serve docs`
4. Open docs in the browser and refresh page

### Expected behavior

Using toc shouldn't result in hydration mismatch

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (24) x64 AMD Ryzen 9 3900X 12-Core Processor
    Memory: 20.34 GB / 31.95 GB
  Binaries:
    Node: 14.18.3 - D:\nodejs\node.EXE
    Yarn: 1.22.17 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 6.14.15 - D:\nodejs\npm.CMD
  Browsers:
    Chrome: 105.0.5195.102
    Edge: Spartan (44.19041.1266.0), Chromium (105.0.1343.27)
    Internet Explorer: 11.0.19041.1566
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by brc-dd** [comment]
I'm unable to reproduce this. Which VitePress version are you using?

**Comment by DawidBlaszczyk** [comment]
I was using version `0.22.4` and didnt notice vitepress released 1.0 version. In the newest `1.0.0-alpha.13` everthing is working fine. Thanks for the assistance.
